build/cloudbuild-lint.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 1200s
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  id: Build forseti test image
+  args: ['build', '--target', 'forseti-test', '-t', 'forseti-security/forseti-test:test', '.']
```

Comment:
will multiple concurrent tests have race condition on the image target?

Reply:
These will all build separately and build an image of the branch that is being pushed. They will not interfere with each other.